cli/main.rs

```diff
@@ -593,21 +423,8 @@ async fn eval_command(
   // Save our fake file into file fetcher cache
   // to allow module access by TS compiler.
   ps.file_fetcher.insert_cached(file);
-  debug!("main_module {}", &main_module);
-  if ps.options.compat() {
-    worker.execute_side_module(&compat::GLOBAL_URL).await?;
-  }
-  worker.execute_main_module(&main_module).await?;
-  worker.dispatch_load_event(&located_script_name!())?;
-  loop {
-    worker.run_event_loop(false).await?;
-
-    if !worker.dispatch_beforeunload_event(&located_script_name!())? {
-      break;
-    }
-  }
-  worker.dispatch_unload_event(&located_script_name!())?;
-  Ok(0)
+  let exit_code = worker.run().await?;
+  Ok(exit_code)
```

Comment:
Change here to include the exit code is mostly a formality. It seems like `deno eval` already returned the exit code.